src/cmd/vm.rs

```diff
@@ -176,69 +178,229 @@ fn enable_kvm() -> Result<()> {
+    /// for betty.sh. Reuse the VM image. It is true by default. If you want to
+    /// disable it, use `--reuse-disk-image false`.
     #[argh(option, default = "true")]
     reuse_disk_image: bool,
 
-    /// start betty with rootfs verification. It is false by default.
+    /// for betty.sh. Start betty with rootfs verification. It is false by
+    /// default.
     #[argh(switch)]
     rootfs_verify: bool,
 
-    /// the ChromeOS version to use (e.g. R72-11268.0.0). Alternatively,
-    /// postsubmit builds since R96-14175.0.0-53101 can also be specified. It is
-    /// the latest version by default.
+    /// for betty.sh. The ChromeOS version to use (e.g. R72-11268.0.0).
+    /// Alternatively, postsubmit builds since R96-14175.0.0-53101 can also
+    /// be specified. It is the latest version by default.
     #[argh(option)]
     version: Option<String>,
 
-    /// path to betty VM image to start. It has priority over --board and
-    /// --version (they will be ignored)
+    /// for betty.sh. Path to betty VM image to start. It has priority over
+    /// --board and --version (they will be ignored)
     #[argh(option)]
     vm_image: Option<String>,
 
-    /// extra arguments to pass to betty.sh. You can pass other options like
-    /// --extra-args "options".
+    /// for acloud. Launch a cloud based VM instance. It is false by default.  
+    #[argh(switch)]
+    acloud: bool,
+
+    /// for acloud. The Android branch. It is required if --acloud is
+    /// specified.
+    #[argh(option)]
+    branch: Option<String>,
+
+    /// for acloud. The Android Build ID. It is required if --acloud is
+    /// specified.
+    #[argh(option)]
+    build_id: Option<String>,
+
+    /// for acloud. Select ARC-container, not ARCVM. It is false by default.
+    #[argh(switch)]
+    container: bool,
+
+    /// extra arguments to pass to betty.sh or acloud. You can pass other
+    /// options like --extra-args "options".
     #[argh(option)]
     extra_args: Option<String>,
 }
 
 fn run_start(args: &ArgsStart) -> Result<()> {
+    if args.acloud {
+        run_acloud(args)?;
+    } else {
+        run_betty_start(args)?;
+
+        println!("To connect the betty instance, run `lium dut shell --dut localhost:9222`.");
+        println!("To push an Android build a betty VM, run `lium arc flash`.");
+    }
+
+    Ok(())
+}
+
+fn run_acloud(args: &ArgsStart) -> Result<()> {
+    let branch = args
+        .branch
+        .clone()
+        .ok_or(anyhow!("--branch option is required when using acloud"))?;
+    let git_branch = format!("git_{branch}");
+
+    let build_id = args
+        .build_id
+        .clone()
+        .ok_or(anyhow!("--build-id option is required when using acloud"))?;
+    let re = regex!(r"^\d+$");
+    if !re.is_match(&build_id) {
+        bail!("--build-id must be a digit.");
+    }
+
+    let config = Config::read()?;
+
+    let cmd_path = config
+        .acloud_cmd_path()
+        .context("Please configure acloud_cmd_path")?;
```

Comment:
isn't this just acloudw? Or is there some othe rpass the installer will set (I don't recall)

Reply:
It is acloudw, so I'll correct function names ect.